plugins/Monolog/config/config.php

```diff
@@ -11,17 +11,24 @@
 
     'Psr\Log\LoggerInterface' => DI\get('Monolog\Logger'),
 
+    'log.handler.classes' => DI\factory(function (ContainerInterface $c) {
+        $classes = array(
+            'file'     => 'Piwik\Plugins\Monolog\Handler\FileHandler',
+            'screen'   => 'Piwik\Plugins\Monolog\Handler\WebNotificationHandler',
+            'database' => 'Piwik\Plugins\Monolog\Handler\DatabaseHandler',
+        );
+
+        return $classes;
+    }),
```

Comment:
This bit can also be simplified in the same way since it's just a value (it's a PHP array):

```php
    'log.handler.classes' => array(...)
```